apps/sensenet/src/utils/deepMerge.ts

```diff
@@ -0,0 +1,27 @@
+import { DeepPartial } from 'redux'
```

Comment:
Would it make sense to move this to sn-client-utils? Don't we import a lib that has a function like this?

Reply:
👍